conf/modules_nanopore.config

```diff
@@ -0,0 +1,373 @@
+                mode: 'copy',
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+                enabled: params.save_reference
+            ]
+        }
+
+        withName: 'MOSDEPTH_GENOME' {
+            ext.args = '--fast-mode'
+            publishDir = [
+                path: { "${params.outdir}/${params.artic_minion_caller}/mosdepth/genome" },
+                mode: 'copy',
+                pattern: "*.summary.txt"
+            ]
+        }
+
+        withName: 'PLOT_MOSDEPTH_REGIONS_GENOME' {
+            ext.args = '--input_suffix .regions.bed.gz'
+            publishDir = [
+                path: { "${params.outdir}/${params.artic_minion_caller}/mosdepth/genome" },
+                mode: 'copy',
+                pattern: "*.{tsv,pdf}"
+            ]
+        }
+
+        withName: 'MOSDEPTH_AMPLICON' {
+            ext.args = '--fast-mode --use-median --thresholds 0,1,10,50,100,500'
+            publishDir = [
+                path: { "${params.outdir}/${params.artic_minion_caller}/mosdepth/amplicon" },
+                mode: 'copy',
+                pattern: "*.summary.txt"
+            ]
+        }
+
+        withName: 'PLOT_MOSDEPTH_REGIONS_AMPLICON' {
+            ext.args = '--input_suffix .regions.bed.gz'
+            publishDir = [
+                path: { "${params.outdir}/${params.artic_minion_caller}/mosdepth/amplicon" },
+                mode: 'copy',
+                pattern: "*.{tsv,pdf}"
+            ]
+        }
+    }
+}
+
+if (!params.skip_pangolin) {
+    process {
+        withName: 'PANGOLIN' {
+            publishDir = [
+                path: { "${params.outdir}/${params.artic_minion_caller}/pangolin" },
+                mode: 'copy',
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            ]
+        }
+    }
+}
+
+if (!params.skip_nextclade) {
+    process {
+        withName: 'UNTAR' {
+            publishDir = [
+                path: { "${params.outdir}/genome" },
+                mode: 'copy',
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+                enabled: params.save_reference
+            ]
+        }
+
+        withName: 'NEXTCLADE_DATASETGET' {
+            publishDir = [
+                path: { "${params.outdir}/genome" },
+                mode: 'copy',
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+                enabled: params.save_reference
+            ]
+        }
+
+        withName: 'NEXTCLADE_RUN' {
+            publishDir = [
+                path: { "${params.outdir}/${params.artic_minion_caller}/nextclade" },
+                mode: 'copy',
+                pattern: "*.csv"
+            ]
+        }
+
+        withName: 'MULTIQC_TSV_NEXTCLADE' {
+            publishDir = [
+                path: { "${params.outdir}/multiqc/${params.artic_minion_caller}" },
```

Comment:
This will be a good example to see if it's being overwritten by `MULTIQC` config selector

Reply:
Good point. I will check anyway but be great if you can double check the alias stuff that came up on #rnaseq won't be a problem here 🤔

Reply:
I think we are good. I ran this command in `subworkflows` and `workflows` and we don't have any aliases that are the same as the process name.

```
find ./ -type f -exec grep -H 'include' {} \; | grep " as "
```

Reply:
I am running a small test to check if in the case of `MULTIQC,` which is the only case where this is happening, could raise any issue. Anyway, I don't think this could be a big issue.

Reply:
You are right! Dont need to change anything :smile: